automation_tools/__init__.py

```diff
@@ -1644,15 +1644,16 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
-def host_pings(host, attempts=200):
+def host_pings(host, timeout=15):
     """This ensures the given IP/hostname pings succesfully.
 
     :param host: A string. The IP or hostname of host.
-    :param attempts: An integer.
-        The maximum number of attempts to ping the host.
+    :param timeout: An integer.
+        The timeout in minutes to ping the host.
 
     """
-    for attempt in range(attempts):
+    timeup = time.time()+(int(timeout)*60)
```

Comment:
Python's operator precedence is the same as in algebra: multiplication and division before addition and subtraction.

```python
>>> 1 + 2 * 10
21
```

Nitpick: Add some whitespace around each operator and drop unnecessary parentheses:

```python
timeup = time.time() + int(timeout) * 60
```